python/pyspark/sql/tests/test_collection.py

```diff
@@ -289,6 +289,28 @@ def check_to_pandas_for_array_of_struct(self, is_arrow_enabled):
         else:
             self.assertEqual(type(pdf["array_struct_col"][0]), list)
 
+    def check_to_pandas_for_empty_df_with_nested_array_columns(self):
```

Comment:
@ueshin Would this test be run for both classic and connect? Or do I need to add a test to `test_connect_collection` as well?